File: ServiceTimeRecord/ServiceTimeRecord/Models/Time.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace ServiceTimeRecord.Models
{
    [Table("Times")]
    public class Time
    {
        [Key]
        public int id { get; set; }
        [Required]
        public DateTime date { get; set; }
        [Required]
        public int hours { get; set; }
        public Task task { get; set; }
        public int taskId { get; set; }
    }
}

File: ViewTimeRecord/src/app/model/time.ts
export class Time {
    constructor(id = null, date = null, hours = null, taskId = null){
        this.id = id,
        this.date = date,
        this.hours = hours,
        this.taskId = taskId
    }
    id: number;
    date: Date;
    hours: number;
    taskId: number;
}
